tutorials/README.md

```diff
@@ -1,9 +1,10 @@
 <a href="https://explosion.ai"><img src="https://explosion.ai/assets/img/logo.svg" width="125" height="125" align="right" /></a>
 
-# 🪐 Project Templates: Tutorials (11)
+# 🪐 Project Templates: Tutorials (12)
 
 | Template | Description |
 | --- | --- |
+| [`dep_low_resource`](dep_low_resource) | Training a POS tagger and dependency parser for a low-resource language |
```

Comment:
Also update this README?

Reply:
Updated in https://github.com/explosion/projects/pull/122/commits/646a868cd9fba78efc480972410eb16b8aaf7338